MobileBuy/sample/src/main/java/com/shopify/sample/application/SampleApplication.java

```diff
@@ -85,6 +89,13 @@ public static void setCustomer(Customer customer) {
     private Checkout checkout;
     private Shop shop;
 
+    private MaskedWallet maskedWallet;
+
+    public static final String ANDROID_PAY_FLOW = "com.shopify.sample.androidpayflow";
+
+    // Use ENVIRONMENT_TEST for testing
+    public static final int WALLET_ENVIRONMENT = WalletConstants.ENVIRONMENT_SANDBOX;
```

Comment:
That works well for plain strings, but not so well where we need to refer to constants defined in a thirdparty java library.
However, I should move this to ENVIRONMENT_TEST here before merging.